### DISCUSSION TITLE
震惊！法外狂徒发来恐怖录像！！！

### DISCUSSION THREAD

**Discussion by ybbgh** [share121/inter-knot#3447] [其他]
他宝了个贝的，伟大的花火大人精心准备的炸弹其实是烟花！而你们却只找到了几个！
伟大的花火大人决定不给你们机会了！他要吟唱咒语！让你们感受人类刻在基因中的最原始的恐惧！！！
听好了！<img src="https://bbs-static.miyoushe.com/static/2024/06/19/8079e2c46c0698178c735198f9340cff_3596969066154449614.png" alt="花火 礼物" width="75" >
https://github.com/user-attachments/assets/c1cb13e6-950d-4529-a586-ecdf76180d20

**Comment by jiangshixie** [comment]
花斯卡慕斯卡小花火~ 花斯卡慕斯卡——小——花——火——（复读）

**Reply by ybbgh** [reply]
是花斯卡，火斯卡！果然只有伟大的花火大人可以正确的吟唱出咒语！
啊！~花火~~~~~~<img src="https://bbs-static.miyoushe.com/static/2024/01/31/cf527198a404a4db566dc4a9d68befc3_2947652818567379105.png" alt="花火 嘿嘿" width="75" >
